test/testlib/nuodb_database_utilities.go

```diff
@@ -423,17 +427,18 @@ func GetNuoDBVersion(t *testing.T, namespaceName string, options *helm.Options)
 		options.SetValues["nuodb.image.tag"])
 	output, err := k8s.RunKubectlAndGetOutputE(
 		t, kubectlOptions, "run", podName,
-		"--image", nuodbImage, "--restart=Never", "--rm", "--attach", "--command", "--",
+		"--image", nuodbImage, "--restart=Never", "--attach", "--command",
+		"--image-pull-policy=IfNotPresent", "--",
 		"nuodb", "--version")
 	require.NoError(t, err, "Unable to check NuoDB version in helper pod")
 	match := regexp.MustCompile("NuoDB Server build (.*)").FindStringSubmatch(output)
 	require.NotNil(t, match, "Unable to match NuoDB version from output")
 	return match[1]
 }
 
-func SkipTestOnNuoDBVersion(t *testing.T, versionCheckFunc func(*semver.Version) bool) {
+func RunOnNuoDBVersion(t *testing.T, versionCheckFunc func(*semver.Version) bool, actionFunc func(*semver.Version)) {
 	randomSuffix := strings.ToLower(random.UniqueId())
-	defer Teardown(TEARDOWN_ADMIN)
+	defer Teardown(TEARDOWN_NGINX)
 	namespaceName := fmt.Sprintf("nuodbversioncheck-%s", randomSuffix)
 	CreateNamespace(t, namespaceName)
```

Comment:
who deletes this namespace? TEARDOWN_ADMIN in the caller?

Reply:
Yeah, the change was temporary and not correct. Please see my comments in the above post